control/statefbk.py

```diff
@@ -61,6 +63,18 @@ def place(A, B, p):
         Input matrix
     p : 1-d list
         Desired eigenvalue locations
+    method : string - optional
```

Comment:
[Numpydoc](https://numpydoc.readthedocs.io) suggests the format "string, optional" to indicate optional arguments and also suggests that the default value be specified in the description of the parameter.